tests/test_outlier.py

```diff
@@ -660,7 +663,7 @@ def test_wrong_info_get_ood_predictions_scores():
 @given(
     fill_value=st.floats(
         min_value=5 * float(np.finfo(np.float_).eps),
-        max_value=10,
+        max_value=5,
```

Comment:
This is a sensible value. The original max_value=10 was arbitrarily chosen. It's nice to be able to test larger values, but the focus is values below the order of 1.